eth/fetcher/tx_fetcher.go

```diff
@@ -262,57 +262,82 @@ func (f *TxFetcher) Notify(peer string, hashes []common.Hash) error {
 // direct request replies. The differentiation is important so the fetcher can
 // re-shedule missing transactions as soon as possible.
 func (f *TxFetcher) Enqueue(peer string, txs []*types.Transaction, direct bool) error {
-	// Keep track of all the propagated transactions
+	var (
+		inMeter, knownMeter, underpricedMeter, otherRejectMeter metrics.Meter
```

Comment:
nitpicks, perhaps it will make eyes more comfortable to define them one by one like 

```go
var (
		inMeter          metrics.Meter
		knownMeter       metrics.Meter
		underpricedMeter metrics.Meter
		otherRejectMeter metrics.Meter
	)
```